<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.hp.demo.server.login.LoginComponentTest" time="0" tests="1" errors="0" skipped="0" failures="1">
  <testcase name="testRegistrationCallback" classname="com.hp.demo.server.login.LoginComponentTest" time="0">
    <failure message="Continuously failing" type="java.lang.AssertionError">
      java.lang.AssertionError: Continuously
      failing
      at org.junit.Assert.fail(Assert.java:88)
      at com.hp.demo.server.login.LoginComponentTest.testRegistrationCallback(LoginComponentTest.java:34)
    </failure>
  </testcase>
</testsuite>